src/components/App/App.tsx

```diff
@@ -166,13 +149,7 @@ class App extends React.Component<Props, State> {
                 >
                   <span className='mr3 o-20 bold'>{index + 1}</span> {chapter.title}
                 </Link>
-                {shouldDisplaySubtitles(
-                  selectedTrackAlias,
-                  index,
-                  chapters,
-                  chapter,
-                  location.pathname.split('/')[1]
-                ) && chapter.subchapters.map((subchapter) => (
+                {(!chapter.isTrack || selectedTrackAlias === chapter.alias) && chapter.subchapters.map((subchapter) => (
```

Comment:
can you please still extract this into a function? it is not immediately clear what this condition expresses. a function name could help with that.